spec/openapi.yaml

```diff
@@ -411,6 +436,46 @@ components:
           description: "The keyword to delete."
     DeleteKeywordResponse:
       type: object
+    ManagedConfigRequest:
+      type: object
+      properties:
+        env_tz_offset:
+          type: string
+          description: The local time offset of the environment that the client is running within.
+        environment:
+          type: object
+          description: A dictionary of client runtime environment properties.
+          additionalProperties:
+            type: string
+          example:
+            java_vm_specification_version: 1.8
+            java_vm_version: 2.8
+            java_vm_name: IBM J9 VM
+            client_os: linux
+        connector_type:
+          type: string
+          description: The paired system type that the connector supplies activity data to.
+        library_version:
+          type: string
+          description: The client library version.
+        client_ts:
+          type: number
+          format: int64
+          description: The system timestamp from epoch of the client environment.
+    ManagedConfigResponse:
+      type: object
+      properties:
+        service_id:
```

Comment:
`service_id` and `service_key` can raise questions on what is this service. Is it intentionally to be able switch underline service type later?